pkg/client/unversioned/clientcmd/overrides.go

```diff
@@ -172,6 +175,7 @@ func BindAuthInfoFlags(authInfo *clientcmdapi.AuthInfo, flags *pflag.FlagSet, fl
 func BindClusterFlags(clusterInfo *clientcmdapi.Cluster, flags *pflag.FlagSet, flagNames ClusterOverrideFlags) {
 	flagNames.APIServer.BindStringFlag(flags, &clusterInfo.Server)
 	flagNames.APIVersion.BindStringFlag(flags, &clusterInfo.APIVersion)
+	flagNames.PreferredVersions.BindStringFlag(flags, &clusterInfo.PreferredVersions)
```

Comment:
You can have `[]string` flags now that would allow `--preferred-version=foo --preferred-version=bar`.  What about `--version`?  Too generic?